src/pages/elements/Namespace.js

```diff
@@ -4,6 +4,19 @@ import GrafanaUrl from './GrafanaUrl';
 import Definition from '../../components/Definition';
 
 function Namespace({ namespace }) {
+  let grafana;
+  if (namespace.grafanaUrl !== null) {
+    grafana = (
+      <a href={namespace.grafanaUrl} target="_blank" rel="noopener noreferrer">
+        Link
+      </a>
+    );
+  } else {
+    grafana = (
+      <GrafanaUrl jumpHost={namespace.cluster.jumpHost} cluster={namespace.cluster.name} namespace={namespace.name} />
```

Comment:
so, I was just looking at the code, and I think we should add the above logic to the GrafanaUrl component.

You could add a new prop: `url` and if set and not null, it will use that one to create the link.

This means that you would only need to add `url={namespace.grafanaUrl}` to every `GrafanaUrl` component, and simply check inside of the component if it's null.

WDYT?

Reply:
Yep I think that would be a better solution, I can fix that up tomorrow!